vowpalwabbit/example_predict.h

```diff
@@ -43,6 +43,7 @@ struct example_predict
   v_array<namespace_index> indices;
   std::array<features, NUM_NAMESPACES> feature_space;  // Groups of feature values.
   uint64_t ft_offset;                                  // An offset for all feature values.
+  uint32_t stack_depth;                                // Used for debugging reductions.  Keeps track of current reduction level
```

Comment:
nit: should be private? since it's only for debugging and to avoid people assuming functionality from the name

Reply:
Changed the name to make usage more clear but left it public.